### PR TITLE
Booster rocket

### PR DESCRIPTION
**Pull request by codinglikeagirl42**
Ready for review.

### CODE REVIEW

File: src/chapters/booster-rockets/personaldrive.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    
+    Apple Computer founder Steve Jobs on attempts to get companies interested in his and Steve Wozniak's personal computer:
+
+        *So we went to Atari and said, "Hey, we've got this amazing thing, even built with some of your parts, and what do you think about funding us? Or we'll give it to you. We just want to do it. Pay our salary, we'll come work for you."* 
+        *And they said, 'No.'* 
+    
+        *So then we went to Hewlett-Packard,* 
+        *and they said, 'Hey, we don't need you. You haven't got through college yet.'* 
+    
+-
```
Comment (gildedgardenia) [inline src/chapters/booster-rockets/personaldrive.rst:53]: Is this dash intentional?

File: src/chapters/booster-rockets/sportsstuff.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+   * - Sport
+     - Quote
+   * - Baseball
+     - "There may be people who have more talent than you, but there's
+       no excuse for anyone to work harder than you do." - *Derek Jeter*
+   
+   * - Basketball
+     - "I've missed more than 9000 shots in my career. I've lost almost 
+       300 games 26 times I've been trusted to take the game winning shot
```
Comment (gildedgardenia) [inline src/chapters/booster-rockets/sportsstuff.rst:20]: Missing period: "300 games. 26 times ...."

**Review by gildedgardenia — APPROVED** [review]
Just a few fixes and then it is ready to go! Great job!

File: src/chapters/booster-rockets/bestpractices.rst [hunk 3]
```diff
@@ -3,35 +3,23 @@ Best Practices: Learning To Code
...
-For years, the education community has debated the best approach for
-*homework*. Should it be scored or not? What is the proper amount? Should
-it be even be assigned? Many reports begin with "Studies have shown…," but
-no universally accepted conclusion has emerged.
+Traits of Good Programmers
+--------------------------
+
+**Curious. Good communicator. Creative. Detail-oriented. Persistent. Problem solver.**
+
+These are some traits that successful computer programmers share. People come to computer programming from all different walks of life and previous job experiences. But the things they have in common are that they like to solve problems, they can both imagine (or invent) the "big picture" as well as pay attention to the "little details", they can express their ideas and listen attentively to the ideas of others, and they are persistent--they don't give up!
```
[the remark this answers is not in the document]
Comment (codinglikeagirl42) [reply src/chapters/booster-rockets/bestpractices.rst:11]: Fixed, I broke this up into many sentences.

File: src/chapters/booster-rockets/boosterrockets.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+know, or whether they took apart computers when they were eleven years old, or even 
+whether they have a computer science degree. It is how hard they will work to find 
+a solution.
+
+What it takes to succeed as you go through this course and beyond in your coding 
+journey is a determination to not give up, to keep trying new things, to keep 
+taking another look at the problem and coming up with another idea for how to solve 
+it, and to accept that this maddening process is not some flaw in you or in the 
+field, but rather that it is the nature of the job itself, and that you can learn 
+to love it. And that's why you will earn the big bucks.
```
[the remark this answers is not in the document]
Comment (codinglikeagirl42) [reply src/chapters/booster-rockets/boosterrockets.rst:32]: Broken into smaller sentences.

File: src/chapters/booster-rockets/brainbreak.rst [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+.. line-block::
+
+    Make your own playlist, or search for a tech/it/coding/homework music compilations on your internet player of choice (i.e., pandora, spotify, youtube). Some LaunchCode favorites to get you started:
+    
+    `Gloria by Laura Branigan <https://www.youtube.com/watch?v=nNEb2k_EmMg>`_
+    `Gonna Fly Now (Theme from 'Rocky') <https://www.youtube.com/watch?v=Qwu5zWTEYVs>`_
+    `I Will Survive by Gloria Gaynor <https://www.youtube.com/watch?v=FHhZPp08s74>`_
+    `Happy by Pharrell Williams <https://www.youtube.com/watch?v=ZbZSe6N_BXs>`_
+    `Can't Stop the Feeling by Justin Timberlake <https://www.youtube.com/watch?v=ru0K8uYEZWw>`_
```
[the remark this answers is not in the document]
Comment (codinglikeagirl42) [reply src/chapters/booster-rockets/brainbreak.rst:37]: Done

File: src/chapters/booster-rockets/brainbreak.rst [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+**Misc** 
+
+.. line-block::
+
+    Excellent article, `Screw motivation, what you need is discipline. <http://www.wisdomination.com/screw-motivation-what-you-need-is-discipline/>`_ 
+    \* Note that there are a few expletives used in it, so if you are sensitive to swearing, you may not wish to read it.
+
+    `Wizard Zines by Julia Evans <https://wizardzines.com>`_
+    `bukola - youtube channel that talks about life as a dev. <https://www.youtube.com/channel/UC-bFgwL_kFKLZA60AiB-CCQ>`_
```
[the remark this answers is not in the document]
Comment (codinglikeagirl42) [reply src/chapters/booster-rockets/brainbreak.rst:84]: Done

File: src/chapters/booster-rockets/personaldrive.rst (hunk 1 above)
Comment (codinglikeagirl42) [reply src/chapters/booster-rockets/personaldrive.rst:53]: Nope, not suppose to be there, deleted.

File: src/chapters/booster-rockets/sportsstuff.rst (hunk 2 above)
Comment (codinglikeagirl42) [reply src/chapters/booster-rockets/sportsstuff.rst]: Added

**Pull request merged**